tests/io/test_partitioned_dataset.py

```diff
@@ -328,7 +372,7 @@ def test_no_partitions(self, tmpdir):
     def test_filepath_arg_warning(self, pds_config, filepath_arg):
         pattern = (
             f"`{filepath_arg}` key must not be specified in the dataset definition as it "
-            f"will be overwritten by partition path"
```

Comment:
We usually keep an `f` prefix  for all string parts even if they aren't formatted (at least this is how PyCharm does it by default)